build.gradle

```diff
@@ -21,6 +21,7 @@ ext {
 
 def dependencyVersions = [
             ida_utils:'323',
+            rest_utils:'330',
```

Comment:
any reason to use rest-utils at the same time as ida-utils? it's the same lib...

Reply:
we shouldn't let the versions drift if they are both required for some reason

Reply:
(I've not reviewed this PR)

Reply:
Agree that we shouldn't let the versions drift.

In this case upgrading security utils to 330 breaks all kinds of things in some quite non-trivial ways. Andy Paine has been working on getting a fix for that, but for now we are keen to commit this code in a non breaking way, (as its a large enough merge as it is and we'd like to avoid long lived branches)

These should get back in sync again soon - just wanted to be able to commit this work before it causes conflicts.

Reply:
Could you add a comment to that effect so people know it's temporary?

Reply:
This has been done. Please see https://github.com/alphagov/verify-hub/commit/d232e6a84ce73effb46e820b89be782b4a8dac4f.